source/atlas-cli-deploy-docker.txt

```diff
@@ -84,12 +78,23 @@ deployment with Docker.
 
          mongosh {connection_string}
 
-      **Example:**
+      **Examples:**
 
-      .. code-block:: sh
+      .. tabs::
+
+         .. tab:: No Authentication
+            :tabid: no-auth
+
+            .. code-block:: sh
 
-         mongosh mongodb://root:root@localhost:27017/?directConnection=true
+               mongosh "mongodb://localhost:27017/?directConnection=true"
 
+         .. tab:: With Authentication
+            :tabid: with-auth
+
+            .. code-block:: sh
+
+               mongosh "mongodb://user:pass@localhost:27017/?directConnection=true"
 
 Create a Local Atlas Deployment with Docker Compose
```

Comment:
I think we should remove the `Use the atlas deployments command to create a local` part